src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -137,13 +137,13 @@ public static object GetUninitializedObjectWithFormatterServices(Type type)
         private static SerializationContext serializationContext;
 
         private static SerializationContext CurrentSerializationContext
-            => serializationContext ?? (serializationContext = new SerializationContext());
+            => serializationContext ?? (serializationContext = new SerializationContext(null));
         
         [ThreadStatic]
         private static DeserializationContext deserializationContext;
 
         private static DeserializationContext CurrentDeserializationContext
-            => deserializationContext ?? (deserializationContext = new DeserializationContext());
+            => deserializationContext ?? (deserializationContext = new DeserializationContext(null));
```

Comment:
Well, type initialization order is the "error prone" part I'm worried about. If this is fixed in the upcoming PR, then I don't mind, it's just that I'd rather make it throw explicitly than having a context that sometimes works and sometimes doesn't